sunpy/instr/goes.py

```diff
@@ -9,15 +9,25 @@
 from itertools import dropwhile
 
 import numpy as np
-import scipy.interpolate as interpolate
+from scipy import interpolate
+from scipy.integrate import trapz
+from scipy.integrate import cumtrapz
+import astropy
+import astropy.units as u
+from astropy.units.quantity import Quantity
```

Comment:
instead of this import can you do `u.Quantity` instead? It should be a find/replace job.